client/src/reducers/cascade.ts

```diff
@@ -1,5 +1,18 @@
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-export default function cascadeReducers(arg: any) {
+import { AnyAction } from "redux";
+import type { RootState } from ".";
+
+export type ReducerFunction = (
+  prevStateForKey: any,
+  action: AnyAction,
+  nexState?: RootState,
+  prevState?: RootState
+) => any;
```

Comment:
I tried using generics here (e.g. 
```
export type ReducerFunction<T> = (
  prevStateForKey: T,
  action: AnyAction,
  nexState?: RootState,
  prevState?: RootState
) => T;
```) but couldn't get the typing to work out in `reducers/index.ts`

Reply:
Maybe this works?

https://github.com/chanzuckerberg/single-cell-explorer/pull/427

Thank you!